microsoft-edge/devtools-guide-chromium/ie-mode/index.md

```diff
@@ -1,24 +1,24 @@
 ---
 description: IE mode and the Microsoft Edge (Chromium) DevTools
-title: Internet Explorer mode and the DevTools
+title: Use the DevTools in Internet Explorer Mode
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 02/12/2021
+ms.date: 09/03/2021
 ms.topic: article
 ms.prod: microsoft-edge
 keywords: microsoft edge, web development, f12 tools, devtools, ie11, internet explorer 11, ie mode
 ---
-# Internet Explorer mode and the DevTools  
+# Use the DevTools in Internet Explorer Mode  
 
 This article describes how Internet Explorer mode \(IE mode\) integrates with the Microsoft Edge \(Chromium\) DevTools.  
 
 ## Understanding IE mode  
 
 IE mode allows enterprises to specify a list of web sites that only work in Internet Explorer 11.  When you navigate to these web sites in Microsoft Edge \(Chromium\), an instance of Internet Explorer 11 runs and renders the site in a tab.  The functionality allows enterprises to manage compatibility with technologies that are currently not compatible with any modern web browsers.  Support for the following technologies is included in IE mode.  
 
-*   IE document modes  
-*   ActiveX controls  
-*   other legacy components  
+*   IE document modes.  
+*   ActiveX controls.  
+*   Other legacy components.  
 
 In IE mode, the rendering process is based on Internet Explorer 11.  The Microsoft Edge \(Chromium\) process manager handles the lifetime of the rendering process.  It is constrained to the lifetime of the tab for a specific site \(or app\).  When a tab renders in IE mode, a badge appears in the address bar for the specific tab.  
```

Comment:
The "badge" is called "the IE mode indicator icon" at page https://docs.microsoft.com/en-us/deployedge/edge-ie-mode-local-site-list -
"The IE mode indicator icon is visible to the left of the address bar and the flyout shows an option that users can toggle to Open the page in Internet Explorer mode next time. This adds the specific page the user is on to the local site list and will automatically open in IE mode for the next 30 days."